### ISSUE TITLE
Customer Reassurance module v.4.1.1 - Absolute URL's problem

### ISSUE THREAD

**Issue by dreamtheme** [PrestaShop/PrestaShop#16124]
In PS 1.7.6.1 with the upgraded version of Customer Reassurance module v.4.1.1 - there are two bugs:
Bug 1:
The submitted images are stored as absolute URL in the Database - example in the ps_psreassurance database under custom_icon field, we find absolute URL like this 
"ps_dev/ps_clean/ps1761/modules/blockreassurance/views/img/img_perso/ic_verified_user_black_36dp_1x.png"

Old version database image: https://ibb.co/txXHQS9

New version database image: https://ibb.co/2W7XFFv

This is a problem , because if you move the installation from one location to another, this will brake the image display in front, and has to be corrected either by re-submitting the images or correct database URL, this is wrong way to do it. In the older 3.x version only the filename is stored in the database and the URL is constructed via the code, which is the right way to do it.

PrestaShop version: 1.7.6.1
PHP version: 7.2

**Comment by khouloudbelguith** [comment]
Hi @dreamtheme,

Thanks for your report.
1. About the first bug, I have the same behavior
I attached a screen record
https://drive.google.com/file/d/1tAgdrM7beaagQAQiRN18q1JskaUqcrpL/view
Ping @Progi1984, @@PrestaShop/prestashop-product-team what do you think of his suggestion?
This behavior is changed after upgrade to 4.1.1.

2. @dreamtheme, about the second bug, I did not manage to reproduce the issue with PS1.7.6.1 and the `blockreassurance` module v4.1.1.
https://drive.google.com/file/d/1mPckUca8LjYZhZcrQjh_S1yg83FP_Jln/view

Thanks!

**Comment by colinegin** [comment]
Hello,

thanks for your feedback !

About the first issue, as this is a bug (this is not the expected behaviour), I think we should fix it but i let @Progi1984 define the level of complexity to resolve the issue.

About the second issue, as @khouloudbelguith cannot reproduce it, i will remove it from the description, so we clearly know what is expected in this issue. @dreamtheme do not hesitate to share more information about your bug or steps to reproduce it. 

Thanks !

**Comment by dreamtheme** [comment]
@colinegin Yeah, forget the second bug, i could not reproduce it on another server, only on one specific so i don't know, if i ever find what it is, i will post another bug.

**Comment by PierreRambaud** [comment]
I confirm this is a bug, we shouldn't insert in the database the path of the image. Part of PrestaShop installation must be controlled by the PrestaShop Core itself, not inside an SQL field.

**Comment by staniek** [comment]
Hi, I do confirm the bug in prestashop 1.7.6.7 and can add one more place: the ps_psreassurance_lang table. Not only server paths are hardcoded on installation but also the server name. So after a **typical** move to a production we loose links in this module as well as the images.
